.idx/dev.nix

```diff
@@ -0,0 +1,68 @@
+# To learn more about how to use Nix to configure your environment
```

Comment:
On the one hand, I'm wondering if we really want to track this file, but on the other, I'm thinking it might open the door to contributing to the project via IDX, and I think that's an interesting idea, what do you think?

Reply:
I think it's good to leave `.idx/*` files in the repository so that users, using Project IDX, don't have to create their own IDX config files and can quickly start contributing using an (already) working environment.